src/mail.sh

```diff
@@ -959,7 +1052,7 @@ function parse_mail_options()
   local commit_count=''
   local short_options='s,t,f,v:,i,l,n,'
   local long_options='send,simulate,to:,cc:,setup,local,global,force,verify,verbose,'
-  long_options+='template::,interactive,no-interactive,list,private,rfc,'
+  long_options+='template::,interactive,no-interactive,list,private,rfc,groups::,create,'
```

Comment:
I did not understand why groups have an optional parameter. What is the idea behind it? I guess if you add documentation, it can be easier to understand your intention about groups and create them.